src/components/base-gallery/SidePanel.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+
+import { DopplerEditorStore } from "../../abstractions/domain/DopplerEditorSettings";
+import { useGetEditorSettings } from "../../queries/editor-settings-queries";
+import { ContentLoading } from "./ContentLoading";
+import { FormattedMessage } from "react-intl";
+
+export const SidePanel = ({
+  setValue,
+  value,
+}: {
+  value: string;
+  setValue: (store: string) => void;
+}) => {
+  const [loading, setLoading] = useState(true);
+  const [stores, setStores] = useState<DopplerEditorStore[]>([]);
+  const editorSettings = useGetEditorSettings();
+
+ // TODO resolve missing dependency for useEffect
```

Comment:
Is this solved? or will it be solved in another PR?

Reply:
not yet, in other PR